R/git.R

```diff
@@ -72,7 +74,76 @@ git_root_directory <- function(dir) {
   if (is.null(root_dir)) {
     stop(sprintf(
       "Can't run import as can't locate git directory from path %s. %s", dir,
-      "Import must be under version control to be run."))
+      "Import must be under version control to be run."
```

Comment:
Is there a good solution to multi line strings in R? This is one approach I've taken here. In `specio` I think I have created a function which strips duplicate whitespace and new lines from a string to get a similar effect. Neither approach I'm super happy with though. I'm interested in what you do?